wasi/build.rs

```diff
@@ -0,0 +1,9 @@
+fn main() -> std::io::Result<()> {
+    println!("cargo:rerun-if-changed=../wit/deps");
+
+    println!("cargo:rerun-if-changed=wit/deps");
+    println!("cargo:rerun-if-changed=wit/deps.lock");
```

Comment:
Should we add this new deps.lock file to the top-level .gitignore?

Reply:
For this particular crate - yes, definitely, that would eliminate redundant diffs on top-level dep updates, good point! But do note, that in general case (e.g. in the root) the lock should most probably be checked in